src/org/zaproxy/zap/extension/httpsessions/ExtensionHttpSessions.java

```diff
@@ -182,6 +185,9 @@ public void hook(ExtensionHook extensionHook) {
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuRemoveSession());
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuAddUserFromSession());
 			extensionHook.getHookMenu().addPopupMenuItem(getPopupMenuItemCopySessionToken());
+			if (Control.getSingleton().getExtensionLoader().getExtension(ExtensionSearch.class).isEnabled()) {
```

Comment:
Sorry, with enabled I meant `!= null` (if not enabled it returns null).

(Note the above `isEnabled` (L176) is not required, this method is not called if the extension is not enabled.)

Reply:
Fair enough